model/project_parser.go

```diff
@@ -95,7 +95,8 @@ type ParserProject struct {
 	// The below fields can be set for the ProjectRef struct on the project page, or in the project config yaml.
 	// Values for the below fields set on this struct will take precedence over the project page and will
 	// be the configs used for a given project during runtime.
-	DeactivatePrevious *bool `yaml:"deactivate_previous" bson:"deactivate_previous,omitempty"`
+	DeactivatePrevious *bool              `yaml:"deactivate_previous"    bson:"deactivate_previous,omitempty"`
```

Comment:
Just noticed the comment above this? Should the other fields (CommitQueueAliases through PatchAliases) be listed here? And buildBaron/TaskAnnotationSettings if we are allowing these to be set from the yaml